cmab/agent.py

```diff
@@ -0,0 +1,33 @@
+from typing import Dict, List
+
+from bandit import Bandit
+
+
+class Agent:
+    __bandit_names = ('spg', 'light_tank', 'heavy_tank', 'medium_tank', 'at_spg')
```

Comment:
Hello Moroz, 

This naming is because the generic problem where an agent (the player) controls a number of bandits(the tanks) which have a series of arms(the actions) is called the Multi-Armed Bandit (MAB) problem where the names for Agent, Bandit and Arms are standard, across such machine learning algorithms, so, I though that I might make this section of the code abstracted from what in specific the Agents, Bandits and arms actually are since anyone who is familiar with the MAB problem can understand the functioning.

I will add comments explaining what each class represents, but if you prefer I will change the naming, just say so.

Thanks!

Reply:
Yes, this makes sense, either completely abstract the ML section or not, I think I will fully name it according to the specific game names according to the above comment